TwoDave.ServerSentEventsParser/Program.cs

```diff
@@ -7,16 +7,38 @@ class Program
     {
         static void Main(string[] args)
         {
-            var input = "event: foo\r\ndata: some data\r\n\r\n";
-            var remainder = "";
-            SseMessage message = new SseMessage();
+            var path = @".\data\test.txt";
 
-            message = Parser.ParseMessage(input, out remainder);
+            try
+            {
+                var remainder = "";
 
-            Console.WriteLine("message.Id = {0}", message.Id);
-            Console.WriteLine("message.Event = {0}", message.Data);
-            Console.WriteLine("message.Data = {0}", message.Event);
-            Console.WriteLine("Remainder = {0}", remainder);
+                SseMessage message = new SseMessage();
+
+                message = Parser.ParseFile(path, out remainder);
```

Comment:
I think next is to display all messages in the stream.

Reply:
Ok. Will do next commit.